Configuration/PyReleaseValidation/README.md

```diff
@@ -43,23 +43,6 @@ The offsets currently in use are:
 * 0.492: Alpaka, full reco with pixel quadruplets
 * 0.496: Alpaka, full reco with pixel triplets
 * 0.5: Pixel tracking only + 0.1
-* 0.501: Patatrack, pixel only quadruplets, on CPU
-* 0.502: Patatrack, pixel only quadruplets, with automatic offload to GPU if available
-* 0.504: Patatrack, pixel only quadruplets, GPU profiling
-* 0.505: Patatrack, pixel only triplets, on CPU
-* 0.506: Patatrack, pixel only triplets, with automatic offload to GPU if available
-* 0.508: Patatrack, pixel only triplets, GPU profiling
-* 0.511: Patatrack, ECAL only, on CPU
```

Comment:
I think we should keep `.511` and `.521` (just remove Patatrack from the names): those are CPU-only using the legacy modules, might still be useful for comparison and validation.

Reply:
For the pixel part technically these run the CPU Patatrack (so legacy until clusters and then ... *non-legacy*). I can restore them moving them to use fully the legacy reco.

Reply:
`.511` and `.521` should be ECAL only and HCAL only, no Pixel.

Reply:
Ah sorry I have misread!